guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,317 @@
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: <>
+
+-->
+
+<!-----
+Conversion notes:
+
+* Docs to Markdown version 1.0β34
+* Sun Mar 05 2023 13:50:07 GMT-0800 (PST)
+* Source doc: [restructure] Payments with Payjoin: A Case Study
+* Tables are currently converted to HTML tables.
+* This document has images: check for >>>>>  gd2md-html alert:  inline image link in generated source and store images to your server. NOTE: Images in exported zip file from Google Docs may not appear in  the same order as they do in your doc. Please check the images!
+
+----->
+
+# Payments with Payjoin: A Case Study
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+## Introduction
+
+Payjoin transactions can be a powerful tool providing privacy, fee-savings and UTXO consolidation benefits to users. They also present challenges and trade-offs due to their synchronous, multi-party nature.
+
+### Payjoin transactions break some of the chain analysis [heuristics](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#motivation) during the course of making a payment. However, real time coordination between the sender & receiver mediated through an endpoint is required to construct them.
+
+The case study:
+
+- Outlines the incentives and objectives for both senders & receivers
+- Analyses the Payjoin UX on both sides and proposes approaches to design Payjoin experiences.
+- Proposes specific user flows and where possible, working designs for the same.
+
+The high-level findings from this case study are that while the setup is more involved for the receiver, little-to-no effort is required per transaction.
+
+The sender on the other hand needs little-to-no setup (apart from using a supporting wallet) and can achieve an identical or near-identical user experience to a regular transaction, depending on their choices and the sophistication of the implementation.
+
+
+### What is a Payjoin?
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin case study/payjoin-bip78-process.png"
+   retina = "assets/images/guide/case-studies/payjoin case study/payjoin-bip78-process@2x.png"
+   alt-text = "Diagram of a BIP-78 payjoin process"
+   width = 800
+   height = 683
+%}
+
+A bitcoin transaction where the sender and receiver construct a transaction together by contributing inputs is called a Payjoin.
+
+It is also called pay-to-endpoint (P2EP) since the coordination is mediated through an endpoint run by the receiver. The endpoint address is communicated through a BIP-21 URI along with the payment address and amount.
+
+In the payjoin process, parties edit, sign and pass iterations of the transaction between each other before a final version is broadcasted. [BIP-78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Respecting_the_minimum_relay_fee_policy) codifies a protocol with 2 iterations, as shown in the visual here.
```

Comment:
```suggestion
In the payjoin process, parties edit, sign and pass iterations of the transaction between each other before a final version is broadcasted. [BIP-78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Respecting_the_minimum_relay_fee_policy) codifies a protocol with one round of interaction beyond address sharing, as shown in the visual here.
```

In studying interactive protocols, I've always seen and have come to expect interaction quantified in "rounds" from round-robin tournament terminology.

Reply:
Someone else (or you) made the point in the google doc few days back. In an extended payjoin scenario where the sender edits/signs and sends it back to receiver to sign/broadcast the tx, will it be considered 1.5 rounds?

Reply:
I'm open to adding this as an alternative phrasing...